asciimath-utils/README.md

```diff
@@ -0,0 +1,7 @@
+## Regenerate JavaScript files
```

Comment:
just a question: why isn’t that included in the default build?

Reply:
Well that's how it was before this change. See ee8f933c7b34d63a1a0944a8ad1eb2e39e96b22e. I added the file to source control now. It's something I've been doing for other modules as well. I think it's sometimes better to cache files in the source, e.g. when you're downloading files, or when you're generating files and it takes a lot of time.

1. Speeds up the build.
1. Makes the build less dependent on online resources. Downloading takes time, and resources could disappear or change. In the asciimath-utils build we could now remove the checksum check.
1. When you have patches you can simply apply them with git, which is easier to maintain. This is not done for asciimath-utils yet (still using the maven-patch-plugin) but we're doing it for some other modules.